models/ipns_test.go

```diff
@@ -71,6 +71,13 @@ func TestIpnsManager_NewEntry(t *testing.T) {
 			if err != nil {
 				t.Fatal(err)
 			}
+			entries, err := im.FindAll()
+			if err != nil {
+				t.Fatal(err)
+			}
+			if len(entries) != 1 {
+				t.Fatal("failed to find correct amount of entries")
+			}
```

Comment:
we should check that the entry actually is the entry we are looking for, ie compare `entries[0]` with the values provided in `tt.args`

Reply:
good idea